plugins/org.jkiss.dbeaver.ext.cubrid.ui/src/org/jkiss/dbeaver/ext/cubrid/ui/internal/CubridMessages.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.ui.internal;
+
+import org.eclipse.osgi.util.NLS;
+
+public class CubridMessages extends NLS {

```

Comment:
Please look for **the usages** of the `NLS` class through our codebase and read its Javadoc. It's not how you're supposed to use it.

1. You must not specify default values for fields
2. You must call the `NLS#initializeMessages` method with appropriate arguments within the static constructor

Reply:
My bad, I forgot to push this file. I will update it soon. Thank you.

Reply:
I have just fixed it. Thank you.

Reply:
Now looks good. Thanks for bearing with my review comments :^)